ShareBook/ShareBook.Service/Book/BookService.cs

```diff
@@ -239,6 +239,15 @@ public bool UserRequestedBook(Guid bookId)
         public override PagedList<Book> Get<TKey>(Expression<Func<Book, bool>> filter, Expression<Func<Book, TKey>> order, int page, int itemsPerPage)
             => base.Get(filter, order, page, itemsPerPage);
 
+        public IList<Book> GetUserDonations(Guid userId)
```

Comment:
Douglas fizemos algo que irá melhorar os includes, deve ser margiado logo , quando for vc ajusta com o novo Get ok ?